src/AdsList.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Grid from "@mui/material/Grid";
+import AdsCards from './AdsCards';
+import aparts from './data';
+
+    const AdsList = (props) => {
+        
+const {filtrAp} = props;
+const {filtrRo} = props;
+const {filtrCur} = props;
+const {filtrFP} = props;
+const {filtrLP} = props;
+
+   
+    const filteredAparts = aparts.filter(apart => {
+        return apart.category.toLowerCase().includes(filtrAp.toLowerCase())
+    
+    });
+
+    const filteredRooms = filteredAparts.filter(apartN => {
```

Comment:
Fix the logic of the filters, filters should be applied together at once instead of such "steps", create a single value using single filter function 

Also, * task, read about and use `useMemo`